task-2/src/components/MovieCard/index.js

```diff
@@ -3,31 +3,45 @@ import { useDispatch } from 'react-redux';
 
 import PropTypes from 'prop-types';
 
-import { removeMovie } from '../../store/actionCreators';
+import { deleteMovie } from '../../store/thunk';
+import { switchSelectedMovie } from '../../store/actionCreators';
 
 import './index.css';
 
-export const MovieCard = ({ title, genres, release_date, show, onClick }) => {
+export const MovieCard = ({ movie, onClick, setOpenModal }) => {
 	const dispatch = useDispatch();
+
+	const onClickFunction = (event) => {
```

Comment:
Consider giving this handler a more meaningful name.

Reply:
Resolved.